style_editor.html

```diff
@@ -10523,6 +10547,8 @@ <h4 class="settings-section-label" title="These settings are useful for people m
 var state_by_checkbox = new Map();
 var body = document.querySelector("body");
 var structuredView;
+var wavlenInput = FIND("WAVLEN_VALUE");
+var myWavLen = new WavLenClass();
 
 /* Settings buttons saved as local storage */
 function getSavedState(buttonState, defaultValue) {
```

Comment:
How about having this function always return the value, and then have  SaveStateBool doing the conversion to bool?

Reply:
That makes sense.
Done.